src/views/v2/commerce/gateway/stripe/container.php

```diff
@@ -30,4 +30,4 @@
 		<?php $this->template( 'gateway/stripe/card-element' ); ?>
 
 	</form>
-</div>
```

Comment:
Is this file being used anywhere? I think it is now being bypassed. Maybe delete it?

Reply:
Haven't worked on this one. I'm not entirely sure. From what I checked it doesn't seem to be called anywhere. @codingmusician do you know if we're loading this one somewhere?

If not, I'll remove it.

Reply:
Yes. It's being used.

Reply:
Thanks!